cargo-dist/src/cli.rs

```diff
@@ -134,6 +134,8 @@ pub enum Commands {
     /// Report on the dynamic libraries used by the built artifacts.
     #[clap(disable_version_flag = true)]
     Linkage(LinkageArgs),
+    /// List all version tags known by dist.
+    ListTags(ListTagsArgs),
```

Comment:
i wonder if we shouldn't make this a tag command with the subcommand list, e.g. dist tag list